src/sentry/tagstore/snuba/backend.py

```diff
@@ -422,17 +424,24 @@ def get_group_seen_values_for_environments(
             ["max", SEEN_COLUMN, "last_seen"],
         ]
 
-        result = snuba.query(
+        result = snuba.aliased_query(
+            dataset=snuba.Dataset.Events,
             start=start,
             end=end,
             groupby=["group_id"],
-            conditions=conditions,
+            conditions=snuba_filters,
             filter_keys=filters,
             aggregations=aggregations,
             referrer="tagstore.get_group_seen_values_for_environments",
         )
 
-        return {issue: fix_tag_value_data(data) for issue, data in six.iteritems(result)}
+        return OrderedDict(
+            (
+                issue["group_id"],
+                fix_tag_value_data(dict(filter(lambda key: key[0] != "group_id", issue.items()))),
```

Comment:
Could you wrap `issue.items()` in a list? That'll help with py3 related things.

Reply:
Wrapped in `six.iteritems`